test/functional/page_objects/dashboard_page.js

```diff
@@ -125,7 +125,9 @@ export function DashboardPageProvider({ getService, getPageObjects }) {
      */
     async onDashboardLandingPage() {
       log.debug(`onDashboardLandingPage`);
-      return await testSubjects.exists('dashboardLandingPage', 5000);
+      return await testSubjects.exists('dashboardLandingPage', {
+        timeout: 5000
```

Comment:
In some other cases,  the `timeout` argument is based on `defaultFindTimeout `: `defaultFindTimeout * 10`.
Is there a reason not to do it here (`defaultFindTimeout/2`) ? It might look weird a bit :)

Reply:
I agree that hard-coded timeouts are not how we should probably be doing this, but I don't personally want to couple these timeout to the `defaultFindTimeout`. In another PR we can add a config option for a more appropriate timeout based on what we can figure out about why 5sec is used here, and 60sec below.